drivers/timer/nrf_rtc_timer.c

```diff
@@ -160,20 +161,21 @@ static void set_absolute_alarm(uint32_t chan, uint32_t abs_val)
 	uint32_t cc_val = abs_val & COUNTER_MAX;
 	uint32_t prev_cc = get_comparator(chan);
 
-
 	do {
 		now = counter();
 
 		/* Handle case when previous event may generate an event.
-		 * It is handled by setting CC to now (that's furtherst future),
+		 * It is handled by setting CC to now (far in the future),
 		 * in case previous event was set for next tick wait for half
 		 * LF tick and clear event that may have been generated.
```

Comment:
> Moreover, I've noticed that bsim behaves differently and busy wait in case of close event must be longer: full tick instead of half tick.

Perhaps it would be good to include this information in the above comment. Right now someone might get confused why for BSim half LF tick is 31 microseconds.